### DISCUSSION TITLE
4.0.0 beta 887570065fa4

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#63] [Announcements]
Blender 4.0.0 Beta, commit [887570065fa4](https://projects.blender.org/blender/blender/commit/887570065fa4).
Build environment:

    Visual Studio 2019, version 16.11.30
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.0.0-beta.2-887570065fa4'>4.0.0 beta 887570065fa4</a>.</em>

**Comment by 1954sailor** [comment]
I can't believe how lucky I am that someone still looks after Win 7 
Thank you
